pkg/virt-handler/node-labeller/cpu_plugin_test.go

```diff
@@ -37,12 +36,6 @@ import (
 	util "kubevirt.io/kubevirt/pkg/virt-handler/node-labeller/util"
 )
 
-var features = []string{"apic", "clflush", "cmov"}
```

Comment:
1570dea4a668c80723a3ca89b4dba6cb0e27e0a6 should be squashed

Reply:
Done.